netsim/ansible/templates/initial/cumulus_nvue.j2

```diff
@@ -21,17 +21,17 @@
             {{ loopback.ipv6 }}: {}
 {% endif %}
         type: loopback
-{% for l in interfaces|default([]) %}
+{% for l in interfaces|default([]) if l.type in ['lan','p2p','stub','svi','lag'] %}
       {{ l.ifname }}:
         link:
 {%  if l.mtu is defined %}
           mtu: {{ l.mtu }}
 {%  endif %}
           state:
             up: {}
-        type: swp
+        type: {{ 'svi' if l.type=='svi' else 'bond' if l.type=='lag' else 'swp' }}
```

Comment:
You're mixing LAG changes with PR titled "VLAN support", but whatever.

Reply:
just letting you know what to expect next...removed